data/promoter.ini

```diff
@@ -4,83 +4,95 @@ to=osg-3.6-%s-contrib
 repotag=osg36
 dvers=el7 el8 el9
 extra_dvers=
+required_keys=OSG-2 OSG-4
 
 [route 3.6-prerelease]
 from=osg-3.6-%s-testing
 to=osg-3.6-%s-prerelease
 repotag=osg36
 dvers=el7 el8 el9
 extra_dvers=
+required_keys=OSG-2 OSG-4
 
 [route 3.6-testing]
 from=osg-3.6-%s-development
 to=osg-3.6-%s-testing
 repotag=osg36
 dvers=el7 el8 el9
 extra_dvers=
+required_keys=OSG-2 OSG-4
 
 [route hcc]
 from=hcc-%s-testing
 to=hcc-%s-release
 repotag=hcc
 dvers=el7
 extra_dvers=el8 el9
+required_keys=HCC HCC-2
 
 [route 3.6-upcoming]
 from=osg-3.6-upcoming-%s-development
 to=osg-3.6-upcoming-%s-testing
 repotag=osg36up
 dvers=el7 el8 el9
 extra_dvers=
+required_keys=OSG-2 OSG-4
 
 [route 3.6-upcoming-prerelease]
 from=osg-3.6-upcoming-%s-testing
 to=osg-3.6-upcoming-%s-prerelease
 repotag=osg36up
 dvers=el7 el8 el9
 extra_dvers=
+required_keys=OSG-2 OSG-4
 
 [route 23-main]
 from=osg-23-main-%s-development
 to=osg-23-main-%s-testing
 repotag=osg23
 dvers=el8 el9
 extra_dvers=
+required_keys=OSG-23-developer
 
 [route 23-main-prerelease]
 from=osg-23-main-%s-testing
 to=osg-23-main-%s-prerelease
 repotag=osg23
 dvers=el8 el9
 extra_dvers=
+required_keys=OSG-23-developer
```

Comment:
required_keys here means the package needs to be signed by that in order to get promoted, and we definitely want them to be signed with the developer key before going into prerelease... it's just that we _also_ need them to be signed with the auto key in order to actually install them from prerelease.

You can have multiple keys in required_keys, but that's an OR not an AND because it was meant to handle distro-specific keys.